src/server/game/Entities/Player/Player.h

```diff
@@ -282,14 +282,15 @@ struct PvPInfo
 
 struct DuelInfo
 {
-    DuelInfo() : initiator(NULL), opponent(NULL), startTimer(0), startTime(0), outOfBound(0), isMounted(false) {}
+    DuelInfo() : initiator(NULL), opponent(NULL), startTimer(0), startTime(0), outOfBound(0), isMounted(false), underDuel(false) {}
 
     Player* initiator;
     Player* opponent;
     time_t startTimer;
     time_t startTime;
     time_t outOfBound;
     bool isMounted;
+    bool underDuel;
```

Comment:
boolean values should be prefixed with `is` (or `are`), in this case `isUnderDuel`

Reply:
Done.